backend/app/pydantic_models/notes.py

```
@@ -1 +1,11 @@
 # do something here bruh
```

Pylint (3.3.7)

[convention] Missing module docstring

(C0114)